app/facades/search_results_facade.rb

```diff
@@ -10,6 +10,23 @@ def med_search_results(name)
     med_and_ndc_hash(results)
   end
 
+  def save_symptoms(med_id)
+    medication = Medication.find(med_id)
+    symptoms = get_symptoms(medication.product_ndc)
+    if !symptoms.nil?
+      symptoms.each do |symptom|
+        new_sym = Symptom.find_or_create_by(description: symptom)
+        MedicationSymptom.create(medication_id: med_id, symptom_id: new_sym.id)
+      end
+    end
+  end
+
+  private
+
+  def get_symptoms(product_ndc)
+    extract_symptoms(product_ndc)
+  end
```

Comment:
if this code is getting the same exact return value as `extract_symptoms`, is there a need to have this separate method? if it's just for the method to be more descriptive, maybe the method on line 30 can be called `get_symptoms`?

Reply:
Thanks for catching this, I think I was just tired and didn't realize it after refactoring some stuff. Fixed!